bitnami/mongodb-sharded/Chart.yaml

```
@@ -36,4 +36,4 @@ maintainers:
 name: mongodb-sharded
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb-sharded
-version: 9.2.5
+version: 9.2.6
```

Please bump minor version instead given you're introducing new parameters:
```suggestion
version: 9.3.0
```